modules/react-components/src/components/input/dynamic-field.tsx

```diff
@@ -167,6 +167,9 @@ export const DynamicField: FunctionComponent<DynamicFieldPropsInterface> = (
      * Prevent submit from being triggered during initial render
      */
     useEffect(() => {
+        if(!update) {
```

Comment:
Shall we fix formatting here?

Reply:
Fixed with 8535eecb987fefadc2bc7058e3a9070e260baf10